### PR TITLE
ECO-776: Shipping expenses Should contain only shipping amounts

### PR DESCRIPTION
**Pull request by tliokos**

### CODE REVIEW

File: src/SprykerEco/Yves/Ratepay/Form/ElvSubForm.php [hunk 1]
```diff
@@ -38,11 +38,11 @@ public function configureOptions(OptionsResolver $resolver)
     /**
      * @deprecated Use `configureOptions()` instead.
      *
-     * @param \Symfony\Component\OptionsResolver\OptionsResolverInterface $resolver
+     * @param \Symfony\Component\OptionsResolver\OptionsResolver $resolver
```
Comment (dereuromark) [inline src/SprykerEco/Yves/Ratepay/Form/ElvSubForm.php]: This change looks invalid and possibly BC breaking.

Comment (tliokos) [reply src/SprykerEco/Yves/Ratepay/Form/ElvSubForm.php]: `OptionsResolverInterface` doesn't exist anymore

Comment (dereuromark) [reply src/SprykerEco/Yves/Ratepay/Form/ElvSubForm.php]: Are you sure?
If you look into core code you always see the interface (20x), but never the concrete class being used.

Comment (tliokos) [reply src/SprykerEco/Yves/Ratepay/Form/ElvSubForm.php]: Let me check it again

Comment (dereuromark) [reply src/SprykerEco/Yves/Ratepay/Form/ElvSubForm.php]: Ah ok: The core already uses both, and it seems your way is the one to go :+1:

Comment (dereuromark) [reply src/SprykerEco/Yves/Ratepay/Form/ElvSubForm.php]: But the deprecated methods should be kept the way they were afaik. See core.
setDefaultOptions with interface, configure without.

**Pull request merged**